### PR TITLE
Update node.js.yml to use animals project and not book

### PR DESCRIPTION
**Pull request by NV-AndreasHahn**
The animal project does not use company registry

### CODE REVIEW

File: .github/workflows/node.js.yml [hunk 1]
```diff
@@ -13,7 +13,7 @@ jobs:
   build:
     defaults:
       run:
-        working-directory: ./books
+        working-directory: ./animals
```
Comment (NV-LukasNuernberger) [inline .github/workflows/node.js.yml:16]: This doesn't fix ur problem, book won't work unless u re-build the package-lock File w/o ur company registry

File: .github/workflows/node.js.yml [hunk 2]
```diff
@@ -32,4 +32,4 @@ jobs:
         cache-dependency-path: ./books/package-lock.json
     - run: npm ci
     - run: npm run build --if-present
-    - run: npm test
+    - run: npm test --passWithNoTests
```
Comment (NV-LukasNuernberger) [inline .github/workflows/node.js.yml:35]: Set this param in the package.json, then it will work

**Review by NV-LukasNuernberger — CHANGES_REQUESTED** [review]
Please fix

File: .github/workflows/node.js.yml (hunk 1 above)
Comment (NV-AndreasHahn) [reply .github/workflows/node.js.yml:16]: I know, but I don't care

File: .github/workflows/node.js.yml (hunk 2 above)
Comment (NV-AndreasHahn) [reply .github/workflows/node.js.yml:35]: I already did, noob

**Pull request merged**